packages/core/src/components/Cards/EditorialCard/ContentHubCard.css

```diff
@@ -0,0 +1,137 @@
+.root {
+  display: flex;
+  width: 100%;
+  font-family: var(--font-avenir);
+  box-sizing: border-box;
+  height: 100%;
+  border: 2px solid transparent;
+}
+
+.root:after {
+  content: '';
+  position: absolute;
+  z-index: 0;
+  width: 100%;
+  height: 100%;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  transition: opacity 200ms ease-in-out;
+  z-index: -1;
+  box-shadow: 0px 0px 5px 0px rgb(216,216,216,1), 0px 0px 15px 0px rgb(0,0,0,0.25);
```

Comment:
Can we use the variable colours for these?

Reply:
They don't match up with any of the variable colors

Reply:
oh ok 👍